provider/datagen/src/registry.rs

```diff
@@ -157,6 +157,7 @@ macro_rules! registry {
 registry!(
     #[cfg(test)]
     icu_dimension::provider::CurrencyEssentialsV1Marker = "currency/essentials@1",
+    #[cfg(feature = "icu_unitsconversion")]
```

Comment:
nit: move this to the correct alphabetic location

Reply:
done: https://github.com/unicode-org/icu4x/pull/4447